samples/suit/smp_transfer/sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi

```diff
@@ -35,11 +38,11 @@
 	#size-cells = < 0x1 >;
 
 	/delete-node/ partition@54000;
-	cpurad_slot0_partition: partition@66000 {
-		reg = < 0x66000 DT_SIZE_K(160) >;
+	cpurad_slot0_partition: partition@54000 {
```

Comment:
I think recovery partition should be located below main partition (to  decrease a risk of ovewrwrirte)